doc/cookbook/source/examples/gaussian_processes/sparse_gaussian_process_regression.rst

```diff
@@ -0,0 +1,49 @@
+==================================
+Sparse Gaussian Process Regression
```

Comment:
@Saurabh7 
Please check out the table at http://www.gaussianprocess.org/gpml/code/matlab/doc/ for choosing right inference method for one  likelihood.